src/seals/testing/envs.py

```diff
@@ -148,10 +154,9 @@ def test_seed(env: gym.Env, env_name: str, deterministic_envs: Iterable[str]) ->
     """
     env.action_space.seed(0)
     actions = [env.action_space.sample() for _ in range(10)]
-
     # With the same seed, should always get the same result
     seeds = env.seed(42)
-    assert isinstance(seeds, list)
+    assert isinstance(seeds, collections.abc.Sequence)
```

Comment:
Why are we doing this? If you want to allow e.g. tuples, you could add it to the instance check. If you want to allow duck typing, checking for the Sequence abc is not a good idea, because most sequence objects won't be registered with collections.abc.Sequence. Actually, all you do is check the length, so all you want is an object implementing the `__len__` method, which you can check by doing a try except.

Reply:
Did it to allow for tuples (even tho gym demands the return type of `env.seed` be a list, for ALE environments it's a tuple). Will add that to the instance check.

Reply:
I guess if you only expect tuples to arise, then what you did is fine, but then you might as well write tuples specifically. Up to you; I'd add a comment explaining why tuples might be expected either way.